packages/core/admin/admin/src/components/CommandPalette/Items.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useCommand } from './context';
+import Item from './Item';
+
+const Items = ({ items, displayOnSearchOnly }) => {
+  const cmd = useCommand();
+
+  return (
+    <>
+      {items.map((item, idx) => {
+        if (item.component) {
+          return React.cloneElement(item.component, { key: idx });
```

Comment:
Hum the component isn't an elementType but an element so I can't do that unless I change how I send the component prop. If I do so then we won't be able to set propos on the compo before passing it.

Reply:
I think if you need to pass props we should re-evaluate how we're approaching this because composability would be better in this way imo. It doesn't look like you need too at the moment so I'd prefer if we just passed it as an elementType like you have the icons and revisit the issue if we need to pass props... wdyt?

Reply:
Sure that works for me